app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -875,10 +874,6 @@ class LuciBuildService {
       processedProperties['is_fusion'] = 'true';
       if (commit.branch != Config.defaultBranch(Config.flutterSlug)) {
         processedProperties.addAll({
-          // For release candidates, let the flutter tool pick the right engine.
```

Comment:
OH - This was "not master"; so did we do this to enable experimental branches as we don't have a merge queue? In which case, content hashing would solve this so long as they don't make engine builds...

Reply:
Hmm.

Experimental branches _do_ make engine builds though. What should we do instead?